kytos/cli/commands/bug_report/api.py

```diff
@@ -0,0 +1,100 @@
+"""Translate cli commands to non-cli code."""
+import re
+import subprocess
+
+
+class BugReportAPI:
+    """Retrieve Kytos environment information.
+
+    Collect system information, python environment, python packages and
+    installed napps to print this report to the user aiming improve bug
+    reports.
+    """
+
+    @classmethod
+    def bug_report(cls, args):  # pylint: disable=unused-argument
+        """Run all bug report."""
+        cls.system_report()
+        cls.python_environment()
+        cls.python_packages_report()
+        cls.kytos_environment_report()
+
+    @classmethod
+    def system_report(cls):
+        """Display system information.
+
+        Print distribution release system information.
+        """
+        print("="*20+'Platform'+"="*20)
+        lsb_release = cls._execute('lsb_release -a 2> /dev/null')
+        uname = cls._execute('uname -a')
+        print('*'*20+'Release information'+'*'*20)
+        print(lsb_release)
+        print('*'*20+'System Information'+'*'*20)
+        print(uname)
+
+    @classmethod
+    def python_environment(cls):
+        """Display python environment report.
+
+        This method shows the path and version of pip and python.
+        """
+        cls._print_path_and_version('python')
+        cls._print_path_and_version('pip')
+
+    @classmethod
+    def _print_path_and_version(cls, package):
+        """Display a package path and version."""
+        print('*'*20+package.title()+'*'*20)
+        path = cls._execute(f'which {package}')
+        version = cls._execute(f'{package} --version')
+        print(f'path={path}, version={version}')
+
+    @classmethod
+    def python_packages_report(cls):
+        """Display all installed packages using pip freeze.
+
+        This method will modify the output and print:
+
+        pypi packages : 'Package Name | Version'
+        git repository package: 'Package Name | Repository | Version'
+        """
+        lines = cls._execute('pip  freeze').split('\n')
+        print('=' * 20 + 'Python Packages' + '=' * 20)
+        for line in lines:
+            if '==' in line:
+                name, version = line.split('==')
+                print(f'{name:<30} | {version:<30}')
+            if line.startswith('-e '):
+                url, name = line.split('#egg=')
+                url = url.replace('-e ', '')
+                repository, hash_number = cls._parse_github_install(url)
+                print(f'{name:<30} | {repository:<30} | {hash_number:<30}')
+
+    @classmethod
+    def _parse_github_install(cls, url):
+        """Parse the repository url and get the github path and commit hash."""
+        pattern = "(github.com:.*).git@(.{8})"
```

Comment:
It cannot parse the repo and version of a kytos project installed from a local clone. When so, the freeze output is, for example: `git://github.com/kytos/kytos.git@0f87413fd60de588dc8f821fb00cebfb8489d2ed`